cliquet/tests/resource/test_filter.py

```diff
@@ -27,20 +27,20 @@ def test_list_can_be_filtered_on_deleted_with_since(self):
 
     def test_filter_on_id_is_supported(self):
         self.patch_known_field.stop()
-        r = self.db.create(self.resource, 'bob', {})
+        r = self.storage.create(self.resource, 'bob', {})
         self.resource.request.GET = {'id': '%s' % r['id']}
         result = self.resource.collection_get()
         self.assertEqual(result['items'][0], r)
 
     def test_list_cannot_be_filtered_on_deleted_without_since(self):
-        r = self.db.create(self.resource, 'bob', {})
-        self.db.delete(self.resource, 'bob', r['id'])
+        r = self.storage.create(self.resource, 'bob', {})
+        self.storage.delete(self.resource, 'bob', r['id'])
         self.resource.request.GET = {'deleted': 'true'}
         result = self.resource.collection_get()
         self.assertEqual(len(result['items']), 0)
 
     def test_filter_works_with_empty_list(self):
-        self.resource.db_kwargs['user_id'] = 'alice'
+        self.resource.storage_kwargs['user_id'] = 'alice'
```

Comment:
You [named it `storage_kw`](https://github.com/mozilla-services/cliquet/pull/245/files#diff-5443aeaa6b1f4c0c732f73d91e160625R80).